setup.py

```diff
@@ -35,6 +35,17 @@
 PY3 = sys.version_info[0] == 3
 
 
+# See LEGAL-362
+def verify_gpl_dependency():
+    if (not os.getenv("GPL_UNIDECODE")
+            and not os.getenv("SLUGIFY_USES_TEXT_UNIDECODE") == "yes"):
+        raise RuntimeError("By default one of Airflow's dependencies installs a GPL "
+                           "dependency (unidecode). To avoid this dependency set "
+                           "SLUGIFY_USES_TEXT_UNIDECODE=yes in your environment when you "
```

Comment:
I think we should set a default - most people won’t care, especially in testing/evaluating Airflow and being able to just `pip install apache-airflow` is a plus.

Having a warning printed about env var (does pip have a way of surfacing these more directly than printing?) is good, but otherwise just default would be my preferred approach.

Reply:
Am I right in thinking that the code will already choose the best option at runtime, and this is just for what is installed at setup time? Might be worth mentioning that in a doc somewhere?

Reply:
@ashb If you have an idea how to do that. Can we bundle python-slugify for example? I tried that more or less but couldnt get it to work. Setting the env variable in setup.py also doesn't work for reasons stated below. 

Obviously I like to set a default and make it easier for the user.

Reply:
Wait - where is slugify even used? I guess it's a couple of levels down? Cos the only use of it in our repo is in `dev/airflow-license` (and if this is the only version then I say we just force the env var to the GPL-version - which ever way round that is).

Is there some other use via another module we pull in?

Reply:
Ah python-nvd3 pulls it in. Right.